cookbook/specs/spec_add_elys.json

```diff
@@ -0,0 +1,2964 @@
+{
+    "proposal": {
+        "title": "Add Specs: EYLS",
+        "description": "Adding new specification support for relaying eyls data on Lava",
+        "specs": [
+            {
+                "index": "EYLS",
+                "name": "eyls testnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDKFULL"
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 0,
+                "blocks_in_finalization_proof": 1,
+                "average_block_time": 4500,
+                "allowed_block_lag_for_qos_sync": 2,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "47500000000"
+                },
+                "api_collections": [
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "rest",
+                            "internal_path": "",
+                            "type": "GET",
+                            "add_on": ""
+                        },
+                        "apis": [
+                            {
+                                "name": "/elys-network/elys/accountedpool/accounted_pool",
+                                "block_parsing": {
+                                    "parser_arg": [
+                                        "latest"
+                                    ],
+                                    "parser_func": "DEFAULT"
+                                },
+                                "compute_units": 10,
+                                "enabled": true,
+                                "category": {
+                                    "deterministic": true,
+                                    "local": false,
+                                    "subscription": false,
+                                    "stateful": 0
+                                },
+                                "extra_compute_units": 0
+                            },
+                            {
+                                "name": "/elys-network/elys/accountedpool/accounted_pool/{ppol_id}",
```

Comment:
typo: ppol -> pool

Reply:
Fixed 

https://github.com/lavanet/lava/pull/1407/commits/a8320fbf46d0904a46ddde41a9574eece3656248

Reply:
I don't see you fixed it